base/fields/functions.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from base import utils
+
+
+# public methods
+def file_path(self, name):
+    """
+    Generic method to give to a FileField or ImageField in it's upload_to
+    parameter.
```

Comment:
Generic method used to give a FileField or ImageField its `upload_to` parameter.